modules/web/js/ballerina/diagram2/views/default/components/nodes/import-declaration-item.jsx

```diff
@@ -42,6 +49,58 @@ export default class importDeclarationItem extends React.Component {
         this.setState({ highlighted: false });
     }
 
+    getImportedPackageName() {
+        const model = this.props.importDec;
+        let importPkgName = model.parent.getPackageName(model);
+        const pkgIdentifier = importPkgName.split('.');
+        const pkgAliasName = pkgIdentifier[pkgIdentifier.length - 1];
+        if (model.getAlias() !== undefined && model.getAlias().value !== pkgAliasName) {
+            importPkgName = importPkgName + ' as ' + model.getAlias().value;
+        }
+        return importPkgName;
+    }
+
+    setEditedSource(value) {
+        const oldNode = this;
```

Comment:
oldNode refers to this react component ?

Reply:
No. Expression editor passes the `this` argument whent it calls the `setterFunc`. See https://github.com/ballerinalang/composer/blob/83f36e6e96fe5537d07b8428ad606dbbe4cfd998/modules/web/js/expression-editor/expression-editor-utils.js#L178